structuredevent-service-cdp/build.gradle

```diff
@@ -17,10 +17,12 @@ dependencies {
     implementation project(':structuredevent-model')
     implementation project(':structuredevent-api-cdp')
     implementation project(':environment-api')
+    implementation project(':core-api')
```

Comment:
Are you sure is it necessary to include core-api here?

Reply:
sadly yes

Reply:
why do we need that? I cannot see a reason (although I am not familiar with with structuredevent module)

Reply:
turned out that this dependency was a leftover from an earlier change hence I removed it